File: 2_4/2task.js
const readline = require('readline');

function Pangram(sentence) {
    const letters = new Set();
    for (let ch of sentence) {
        if (/[a-zA-Z]/.test(ch)) {
            letters.add(ch);
        }
    }
    return letters.size === 26;
}

const rl = readline.createInterface( {
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter a string of lowercase Latin letters: ", function(sentence) {
    console.log(Pangram(sentence));
    rl.close();
});

File: 2_4/2task.rs
use std::collections::HashSet;
use std::io;

fn pangram(sentence: &str) -> bool {
    let mut letters: HashSet<char> = HashSet::new();
    for ch in sentence.chars() {
        if ch.is_alphabetic() {
            letters.insert(ch);
        }
    }
    letters.len() == 26
}

fn main() {
    let mut sentence = String::new();
    println!("Enter a string of lowercase Latin letters: ");
    io::stdin().read_line(&mut sentence).expect("Failed to read input");
    println!("{}", pangram(&sentence.trim()));
}